typescript/alert-center/src/frontend.ts

```diff
@@ -0,0 +1,69 @@
+import { ArgOptional, GetApi, HandlerContext, PostApi } from "@dbos-inc/dbos-sdk";
+import path from 'path';
+import { Liquid } from "liquidjs";
+import { RespondUtilities, AlertStatus } from "./utilities";
+import { Respondment, AlertEmployeeInfo } from "./operations";
+
+const engine = new Liquid({
+  root: path.resolve(__dirname, '..', 'public'),
+  extname: ".liquid"
+});
+
+async function render(file: string, ctx?: object): Promise<string> {
+  return await engine.renderFile(file, ctx) as string;
+}
+
+export class Frontend {
+
+  @GetApi('/')
+  static frontend(_ctxt: HandlerContext) {
+    return render("app.html", {});
+  }
+
+  @GetApi('/assignment')
+  static async getAssignment(ctxt: HandlerContext, name: string, @ArgOptional more_time: boolean | undefined) {
+    const userRecWF = await ctxt.startWorkflow(Respondment).userAssignmentWorkflow(name, more_time);
```

Comment:
I think the current demo is great, but a bit complicated for demonstrating our Kafka use case. We want to focus on Kafka event processing. The `/respond/more_time` idea is cute, but I think it'd be much simpler if we replace it with a `/respond/wont_fix`.

The  `more_time=true` makes the  userAssignmentWorkflow too complex.

Reply:
Right now it performs a `/respond/cancel` if you log out while you have an assignment. The assignment becomes instantly available for others to take.

Reply:
As discussed, not removing this because it's not clear how to replace it (/cancel would result in nearly immediate re-assignment). And the button adds much great drama for only a few extra lines of code.

But I did rewrite this to make it much easier to read. Previously we had generated "expiration time" in one function and passed it to another. There were also extraneous if-branches that weren't needed. See if you like what we have now?